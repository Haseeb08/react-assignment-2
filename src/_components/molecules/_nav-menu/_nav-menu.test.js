import React from "react";
import { create } from "react-test-renderer";
import {
  render,
  screen,
  getByRole,
  getByPlaceholderText,
  fireEvent,
} from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import NavMenu from "./_nav-menu";
import { BrowserRouter } from "react-router-dom";

describe("Add Image box testing", () => {
  test("Matches the snapshot", () => {
    const input = create(<BrowserRouter><NavMenu /></BrowserRouter>);
    expect(input.toJSON()).toMatchSnapshot();
  });
});
