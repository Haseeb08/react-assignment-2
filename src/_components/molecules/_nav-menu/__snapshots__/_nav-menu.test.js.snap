// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Image box testing Matches the snapshot 1`] = `
<div
  className="makeStyles-menuBox-1"
>
  <a
    className="makeStyles-link-2"
    href="/home"
    onClick={[Function]}
  >
    <div
      className="makeStyles-itemBox-3"
      data-testid="input"
    >
      <h4
        className="MuiTypography-root MuiTypography-h4"
      >
        Home
      </h4>
    </div>
  </a>
  <a
    className="makeStyles-link-2"
    href="/gallery"
    onClick={[Function]}
  >
    <div
      className="makeStyles-itemBox-3"
      data-testid="input"
    >
      <h4
        className="MuiTypography-root MuiTypography-h4"
      >
        Gallery
      </h4>
    </div>
  </a>
  <a
    className="makeStyles-link-2"
    href="/gallery-admin"
    onClick={[Function]}
  >
    <div
      className="makeStyles-itemBox-3"
      data-testid="input"
    >
      <h4
        className="MuiTypography-root MuiTypography-h4"
      >
        Admin
      </h4>
    </div>
  </a>
  <a
    className="makeStyles-link-2"
    href="/contact"
    onClick={[Function]}
  >
    <div
      className="makeStyles-itemBox-3"
      data-testid="input"
    >
      <h4
        className="MuiTypography-root MuiTypography-h4"
      >
        Contact
      </h4>
    </div>
  </a>
  <a
    className="makeStyles-link-2"
    href="/about"
    onClick={[Function]}
  >
    <div
      className="makeStyles-itemBox-3"
      data-testid="input"
    >
      <h4
        className="MuiTypography-root MuiTypography-h4"
      >
        About
      </h4>
    </div>
  </a>
</div>
`;
